src/index.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+require('regenerator-runtime/runtime');
+require('source-map-support').install();
+const Once = require('./utils/Once');
+const _ = require('lodash');
+const bugsnag = require('bugsnag');
+const uuidv1 = require('uuid/v1');
+const LambdaContext = require('./utils/LambdaContext');
+const Config = require('./Config');
+
+module.exports = {
+  initialize: function() {
+    this.once = new Once(async () => {
+      // Set up configuration dict
+      this.config = new Config([
+        require('./DataSources/Environment'),
+        require('./DataSources/Credstash'),
+        require('./DataSources/Clients'),
+        require('./DataSources/Logging'),
+        require('./DataSources/Replacer')
+      ]);
+
+      // Needed until we are only async/await
+      await this.config.legacy().initialize();
+
+      // Set up bugsnag
+      const options = {
+        releaseStage: LambdaContext.stage,
+        sendCode: true,
+        metaData: {
+          revision: process.env.BB_COMMIT,
+          errorId: uuidv1()
+        },
+        filters: ['cvv', 'lastName', 'address1', 'address2', 'address3', 'address4', 'email', 'token',
+        'city', 'state', 'province', 'zip', 'phone', 'birth_month', 'birth_day', 'birth_year',
+        'signature', 'accountNumber', 'routingNumber', 'ssn']
+      };
+      for (let x of process.listeners('uncaughtException')) {
+        process.removeListener('uncaughtException', x);
```

Comment:
This is interesting to me -- is the intention to remove the native Lambda handler?

Reply:
Got it, sounds good